cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -557,26 +596,18 @@ func secureConnectToTPM(ekcfile string) (*secboot.TPMConnection, error) {
 	return secbootSecureConnectToDefaultTPM(ekCertReader, nil)
 }
 
-func insecureConnectToTPM(ekcfile string) (*secboot.TPMConnection, error) {
+func insecureConnectToTPM() (*secboot.TPMConnection, error) {
 	return secbootConnectToDefaultTPM()
 }
 
-// unlockEncryptedPartition unseals the keyfile and opens an encrypted device.
-func unlockEncryptedPartition(name, device, keyfile, ekcfile, pinfile string) error {
-	// TODO:UC20: use secureConnectToTPM if we decide there's benefit in doing that or we
-	//            have a hard requirement for a valid EK cert chain for every boot (ie, panic
-	//            if there isn't one). But we can't do that as long as we need to download
-	//            intermediate certs from the manufacturer.
-	tpm, err := insecureConnectToTPM(ekcfile)
-	if err != nil {
-		return fmt.Errorf("cannot open TPM connection: %v", err)
-	}
-	defer tpm.Close()
+var unlockEncryptedPartition = unlockEncryptedPartitionImpl
```

Comment:
I'm not sure I understand the win in mocking this one vs just keeping mocking secbootActivateVolumeWithTPMSealedKey directly only

Reply:
Agree, I see no point there because the refactoring in this PR reduced unlockEncryptedPartition() to a thin wrapper. Changed to mock secbootActivateVolumeWithTPMSealedKey directly.